conf/search.conf.part

```diff
@@ -294,10 +294,10 @@ source src_zipcode : src_swisssearch
         , quadindex(the_geom) as geom_quadindex \
         , box2d(the_geom) as geom_st_box2d\
         , 1 as rank \
-        , st_y(ST_PointOnSurface(the_geom)) AS x \
-        , st_x(ST_PointOnSurface(the_geom)) AS y \
-        , st_y(st_transform(ST_PointOnSurface(the_geom),4326)) as lat \
-        , st_x(st_transform(ST_PointOnSurface(the_geom),4326)) as lon \
+        , label_y AS x \
+        , label_x AS y \
```

Comment:
yes, but the label was always inversed, it is in surveying style, x=north, y=east. we have the same for all indices. i would also prefer to use x=east, y=north.